src/styles/blocks/testimonials.scss

```
@@ -0,0 +1,56 @@
+.testimonials {
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    background-image: url(../images/quotes.svg);
+    background-repeat: no-repeat;
+    background-position: top;
+
+  }
+
+  &__photo {
+    width: 62px;
+    height: 62px;
+    margin: 30px 0 20px;
+    transition: 0.5s;
```

better write it in one line, please change all over the project.
```suggestion
    transition: transform 0.5s;
```

---

Спасибо, исправила по всем пунктам!)